utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\Packagist\PackageProvider;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * Packages that are not released - old packages or forks
+     * @var string[]
+     */
+    private const EXCLUDED_PACKAGES = [
```

Comment:
I added the note about what package are you tagging, so I think that is safe because you'll know what package are you tagging and can add it to the excluded ones, if you don't want to tag it.

Reply:
@DavidOstrozlik I do agree that it's safe to use and it's a consistent approach in the releaser. But the list of packages is currently in more than one place (eg. `monorepo.md`, `.ci/monorepo_functions.sh`, or `TravisStatusReporter`). This will create problems for us in the future imo and is a clear violation of DRY.

Please, create an issue explaining the situation so we'll fix it eventually.

Reply:
I added #1269  for this.